frontend/www/js/omegaup/problem/collection_list.ts

```diff
@@ -58,17 +65,23 @@ OmegaUp.on('ready', () => {
           tagsList: payload.tagsList,
           sortOrder: sortOrder,
           columnName: columnName,
+          difficulty: payload.difficulty,
         },
         on: {
           'apply-filter': (
             columnName: string,
             sortOrder: omegaup.SortOrder,
+            difficultyParameter: string,
           ): void => {
+            let difficulty: string = changeDifficultyParameter(
```

Comment:
Este valor nunca cambia, así que es buen candidato para ser constante:
```suggestion
            const difficulty: string = changeDifficultyParameter(
```